docs/attributes-registry/http.md

```diff
@@ -67,11 +67,24 @@ SHOULD include the [application root](/docs/http/http-spans.md#http-server-defin
 <!-- semconv attributes.http.deprecated(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
+| `http.flavor` | string | Deprecated, use `network.procol.name` instead. | `1.0` |
```

Comment:
There is a misspelling `procol` instead of `protocol`

Reply:
thanks for catching this! I'll send a fix